rosdep/python.yaml

```
@@ -4334,7 +4334,48 @@ python-rpi.gpio-pip:
   ubuntu:
     pip:
       packages: [RPi.GPIO]
-python-rrdtool:
+python3-rpi.gpio:
+  debian:
+    buster: [python3-rpi.gpio]
+    jessie:
+      pip:
+        packages: [RPi.GPIO]
+    stretch:
+      pip:
+        packages: [RPi.GPIO]
+  ubuntu:
```

This package is available for Fedora as well: https://src.fedoraproject.org/rpms/python-rpi-gpio

```suggestion
  fedora: [python3-RPi.GPIO]
  ubuntu:
```